test/default.py

```diff
@@ -1 +1,25 @@
-print("DEFAULT")
+import os
+
+
+def clean_up() -> None:
+    print("CLEAN UP")
+    print(os.listdir())
+
+
+def pre_build() -> None:
+    print("HELLO")
+    print(os.listdir())
+
+
+def post_install() -> None:
```

Comment:
Could we have one or two args? At least we need the path of the folder we just installed to, so I'm able to do the renaming of folders in the target location.

Reply:
I've added a `BabContext` object that provides the “intended working directory” (i.e., what the working directory should be):
```py
from bpy_addon_build.api import BabContext

def main(ctx: BabContext) -> None:
	print(ctx.current_path)
```